### DISCUSSION TITLE
Add focus-follows-mouse option

### DISCUSSION THREAD

**Discussion by adiknoth** [rxhanson/Rectangle#587] [Ideas]
Hi!

Rectangle is awesome. With the windows nicely tiled next to each other, having to click into a window to focus it becomes unnecessary.

I hereby propose an option that would focus the window as you move in the cursor, maybe with or without a delay (not sure if necessary, would need to experiment).

The code can be taken from https://github.com/sbmpost/AutoRaise, which implements exactly this.

**Comment by rxhanson** [comment]
Thanks for the idea. I don't have a desire to add this in, but if someone creates a Swift PR and has this toggleable via terminal command in Rectangle then I would consider it.
